ebml.xml

```diff
@@ -0,0 +1,61 @@
+		<documentation lang="en" purpose="definition">The minimum DocType version an interpreter has to support to read this file.</documentation>
+	</element>
+
+	<element name="SignatureSlot" path="\EBML\SignatureSlot" id="0x1B538667" type="master">
+		<documentation lang="en" purpose="definition">Contain signature of some (coming) elements in the stream.</documentation>
+	</element>
+	<element name="SignatureAlgo" path="\EBML\SignatureSlot\SignatureAlgo" id="0x7E8A" type="uinteger">
+		<documentation lang="en" purpose="definition">Signature algorithm used (1=RSA, 2=elliptic).</documentation>
+	</element>
+	<element name="SignatureHash" path="\EBML\SignatureSlot\SignatureHash" id="0x7E9A" type="uinteger">
+		<documentation lang="en" purpose="definition">Hash algorithm used (1=SHA1-160, 2=MD5).</documentation>
+	</element>
+	<element name="SignaturePublicKey" path="\EBML\SignatureSlot\SignaturePublicKey" id="0x7EA5" type="binary">
+		<documentation lang="en" purpose="definition">The public key to use with the algorithm (in the case of a PKI-based signature).</documentation>
+	</element>
+	<element name="Signature" path="\EBML\SignatureSlot\Signature" id="0x7EB5" type="binary">
+		<documentation lang="en" purpose="definition">The signature of the data (until a new.</documentation>
+	</element>
+
+	<element name="SignatureElements" path="\EBML\SignatureElements" id="0x7E5B" type="master">
+		<documentation lang="en" purpose="definition">Contains elements that will be used to compute the signature.</documentation>
+	</element>
+	<element name="SignatureElementList" path="\EBML\SignatureElements\SignatureElementList" id="0x7E7B" type="master">
+		<documentation lang="en" purpose="definition">
+			A list consists of a number of consecutive elements that represent one case where data is used in signature. Ex: Cluster|Block|BlockAdditional means that the BlockAdditional of all Blocks in all Clusters is used for encryption.
+		</documentation>
+	</element>
+	<element name="SignedElement" path="\EBML\SignatureElements\SignatureElementList\SignedElement" id="0x6532" type="binary" >
+		<documentation lang="en" purpose="definition">An element ID whose data will be used to compute the signature.</documentation>
+	</element>
+
+	<element name="Void" path="\(-\)Void" id="0xEC" type="binary">
+		<documentation lang="en" purpose="definition">Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.</documentation>
+	</element>
+	<element name="CRC-32" path="\(1-\)CRC-32" id="0xBF" type="binary">
+		<documentation lang="en" purpose="definition">The CRC is computed on all the data of the Master element it's in. The CRC element should be the first in it's parent master for easier reading. All level 1 elements should include a CRC-32. The CRC in use is the IEEE CRC32 Little Endian.</documentation>
+	</element>
+</EBMLSchema>
```

Comment:
Git doesn't like files that don't end up with an empty line. So you should add one to make it look nicer.

Reply:
ok